.github/workflows/analysis.yml

```diff
@@ -6,6 +6,11 @@ on:
       - 'main'
```

Comment:
This workflow runs on `main` and anything that starts with `analysis`. We probably only want the website to update from the `main` branch, so can we run it but not upload if it's running an `analysis` branch?